content/articles/subscription-renewals.markdown

```diff
@@ -0,0 +1,55 @@
+---
+title: Subscription renewals
```

Comment:
Should it be singular?

Reply:
No strong opinion on this one. In my head this is plural like _Subscriptions_...
I say we delegate the decision to a native 🇺🇸 speaker.

Reply:
I'd probably leave it plural. The article is about the topic, and plural generally represents the concept

Reply:
Should be plural